core/src/main/java/org/axonframework/eventhandling/saga/AbstractSagaManager.java

```diff
@@ -186,4 +191,26 @@ public boolean supportsReset() {
     public void performReset() {
         throw new ResetNotSupportedException("Sagas do no support resetting tokens");
     }
+
+    @Override
+    public void send(Message<?> message, ScopeDescriptor scopeDescription) throws Exception {
+        if (!(message instanceof EventMessage)) {
+            String exceptionMessage = String.format(
+                    "Something else than an EventMessage was scheduled for Saga of type [%s], whilst Sagas can only handle EventMessages.",
+                    getSagaType()
+            );
+            throw new IllegalArgumentException(exceptionMessage);
+        }
+
+        if (scopeDescription instanceof SagaDescriptor) {
+            sagaRepository.load(((SagaDescriptor) scopeDescription).getIdentifier().toString())
```

Comment:
note that `sagaRepository.load()` can return `null`, for example when a Saga was ended before the deadline occurred. In that case, it should silently continue (and write a little debug level log entry)